rasa_nlu/config.py

```diff
@@ -89,13 +94,13 @@ def __len__(self):
         return len(self.__dict__)
 
     def items(self):
-        return self.__dict__.items()
+        return list(self.__dict__.items())
 
     def view(self):
         return json.dumps(self.__dict__, indent=4)
 
     def format_env_vars(self, env_vars):
-        keys = [key for key in env_vars.keys() if "RASA" in key]
+        keys = [key for key in list(env_vars.keys()) if "RASA" in key]
```

Comment:
just a question: in py3 .keys() and .items() return iterators, right? so why do we have to convert to a list if we're in a list comprehension?